src/Symfony/Component/Mime/Part/TextPart.php

```diff
@@ -111,8 +113,8 @@ public function bodyToString(): string
 
     public function bodyToIterable(): iterable
     {
-        if (\is_resource($this->body)) {
-            if (stream_get_meta_data($this->body)['seekable'] ?? false) {
+        if (null !== $this->seekable) {
+            if ($this->seekable) {
                 rewind($this->body);
             }
             yield from $this->getEncoder()->encodeByteStream($this->body);
```

Comment:
> I'm pretty sure that the rewind calls are not there by mistake.

this is the only place where `encodeByteStream()` is called, and as you can see, the stream is already rewound.